server/utils/applications/utils.ts

```diff
@@ -206,7 +206,7 @@ const isInapplicable = (application: Application): boolean => {
   return false
 }
 
-const firstPageOfApplicationJourney = (application: Application) => {
+const firstPageOfApplicationJourneyBasedOnTier = (application: Application) => {
```

Comment:
I think `tierQualificationPage` would be clearer. 
Also this function should be tested